core_lib/src/managers/layermanager.cpp

```diff
@@ -180,7 +181,7 @@ LayerBitmap* LayerManager::createBitmapLayer(const QString& strLayerName)
     const QString& name = nameSuggestLayer(strLayerName);
     layer->setName(name);
 
-    Q_EMIT layerCountChanged(count());
+    setNewLayerAsSelected();
 
```

Comment:
Probably not here. `load()` is called after a Project is loaded. It looks a bit weird to see aything related to NewLayer here.

Reply:
It's not being called in load(), github is showing a slightly confusing image.. the method is only added in createLayer methods.

Reply:
Ah, got it. sorry my bad.